contributing/submitting-pr.md

```diff
@@ -202,7 +247,49 @@ Then ensure the menu displays the original repository (`ploomber/PROJECT_NAME`)
 base repository: ploomber/PROJECT_NAME base: main (or master) ⬅️ head repository: YOURUSERNAME/PROJECT_NAME compare: YOURBRANCH
 ```
 
-Then click on open Pull Request.
+Then click on open **Draft Pull Request.** Opening a PR will trigger building
+documentation and running the tests. Wait for a few minutes (most of our tests run in
+<15 minutes). If the checks failed:
+
+![](../assets/checks-failed.png)
+
+Check the logs and fix them (see the [troubleshooting](troubleshooting.md) guide
+for tips). If you're unable to fix the issues after spending some time on them. You
+can message us on Slack so we can help you.
```

Comment:
If you're unable to fix the issues after spending some time on them. You
can message us on Slack so we can help you.

I think it should be one sentence